tools/webhook_test.py

```diff
@@ -258,6 +317,10 @@ def get_and_validate_team():
     else:
         print "Egg level invalid. Assuming level 5"
 elif type == whtypes["5"]:
+    print "Enter cache file name to verify the gym (default: manager_0)\n>",
+    cache_or_invalid()
```

Comment:
This forces the user to use file caching, while this is an optional feature.
You aren't able to send a raid webhook without file caching, so it's better to have this as optional as well.